v2/thirdparty/custom-ui/multi-tenancy.mdx

```diff
@@ -0,0 +1,33 @@
+---
+id: multi-tenancy
+title: "Multi-tenancy Implementation"
+hide_title: true
+---
+# Multi-tenancy Implementation
+
+## What is multi-tenancy? and, Possible flows
+
+diagram
+
+- All tenants on Single Domain
+- Each tenant on a different sub-domain
+  - Handling callbacks through common domain
```

Comment:
these should not be here. This page should contain the simplest setup instructions. Nothing else